pymc3/tests/test_distributions.py

```diff
@@ -1944,7 +1944,6 @@ def test_matrixnormal(self, n):
     @pytest.mark.parametrize("n", [2, 3])
     @pytest.mark.parametrize("m", [3])
     @pytest.mark.parametrize("sigma", [None, 1.0])
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
     def test_kroneckernormal(self, n, m, sigma):
         np.random.seed(5)
         N = n * m
```

Comment:
@ricardoV94 mentioned that we aren't supporting broadcasting means of multivariate distributions anymore. 

To me, this seems to be the case in the current test configuration. (Different `n` and `m` generate `covs` and `chols` matrices of different dimensions, and the domain is a vector of their product) But I'd like someone to confirm weather the current parameters of the test are still valid or weather they should be changed.

Reply:
We are supporting "some" cases, but not as many as in `v3`. There was a discussion about this in the port of the `MultivariateT` here: https://github.com/pymc-devs/pymc3/pull/4731

Reply:
Thanks for the tip, I think I'm facing a case in `MatrixNormal` in #4777, where like `MVStudentT` the sample output is not scalar but matrix dependent on the parameters.

Reply:
> We are supporting "some" cases, but not as many as in `v3`. There was a discussion about this in the port of the `MultivariateT` here: #4731

How do you mean?  We should be able to support _all_ well-defined forms of broadcasting, and with a lot less effort than it took in `v3`.

Reply:
I guess it falls under the not well-defined category. Some cases like `pm.MvNormal.dist(0, np.eye(3), shape=(3,3))`, which would work in `V3` but not in `V4`.

Reply:
> I guess it falls under the not well-defined category. Some cases like `pm.MvNormal.dist(0, np.eye(3), shape=(3,3))`, which would work in `V3` but not in `V4`.

Yes, that is a mathematically misspecified distribution, and the problem isn't with broadcasting&mdash;the problem is that `v3` was incorrectly changing the user input to something that might make sense, instead of raising an error like it should have.

Reply:
I used `*covs` as @brandonwillard suggested and that handles the differently shaped matrices in the arguments. So I think we're good over here.